pkg/util/bandwidth/linux.go

```diff
@@ -128,20 +128,21 @@ func asciiCIDR(cidr string) (string, error) {
 	return fmt.Sprintf("%s/%d", ip.String(), size), nil
 }
 
-func (t *tcShaper) findCIDRClass(cidr string) (class, handle string, found bool, err error) {
+func (t *tcShaper) findCIDRClass(cidr string) (classAndHandle [][]string, found bool, err error) {
 	data, err := t.e.Command("tc", "filter", "show", "dev", t.iface).CombinedOutput()
 	if err != nil {
-		return "", "", false, err
+		return [][]string{}, false, err
 	}
 
 	hex, err := hexCIDR(cidr)
 	if err != nil {
-		return "", "", false, err
+		return [][]string{}, false, err
 	}
 	spec := fmt.Sprintf("match %s", hex)
 
 	scanner := bufio.NewScanner(bytes.NewBuffer(data))
 	filter := ""
+	result := [][]string{}
```

Comment:
Declare this in L132. And return `result`, instead of `[][]string{}` every time.

Reply:
It's been modified as you said.